prow/tide/tide.go

```diff
@@ -380,10 +380,10 @@ func (c *Controller) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer c.m.Unlock()
 	b, err := json.Marshal(c.pools)
 	if err != nil {
-		c.logger.WithError(err).Error("Decoding JSON.")
+		c.logger.WithError(err).Error("Encoding JSON.")
 		b = []byte("[]")
 	}
-	fmt.Fprintf(w, string(b))
+	fmt.Fprint(w, string(b))
```

Comment:
why not w.Write(b) ?

Reply:
I think they should be functionally equivalent. `w.Write(b)` is probably a little faster though.

Fixed and added an error check.

Reply:
yeah, the output here would be the same but it's costlier on top of the unnecessary string conversion.